policy/modules/system/systemd.te

```diff
@@ -1547,12 +1572,17 @@ systemd_dbus_chat_logind(systemd_user_runtime_dir_t)
 seutil_read_file_contexts(systemd_user_runtime_dir_t)
 seutil_libselinux_linked(systemd_user_runtime_dir_t)
 
+userdom_manage_user_tmp_dirs(systemd_user_runtime_dir_t)
+userdom_manage_user_tmp_files(systemd_user_runtime_dir_t)
+userdom_manage_user_tmp_pipes(systemd_user_runtime_dir_t)
+
```

Comment:
what pipes are you referring to here? all systemd_user_runtime_dir_t should be doing it create /run/user/* and mount a tmpfs on it. It shouldnt be creating anything below /run/user/*/ AFAIK

Reply:
I had a denial for these at some point, but I'm not able to reproduce it. I will drop this access for now.